site/guides/integration-testing.adoc

```diff
@@ -0,0 +1,970 @@
+    <artifactId>maven-failsafe-plugin</artifactId>
+    <executions>
+      <execution>
+        <goals>
+          <goal>integration-test</goal>
+          <goal>verify</goal>
+        </goals>
+      </execution>
+    </executions>
+  </plugin>
+----
+
+We need to take care however to disable the plugin in the `tests` module like so:
+
+[source,xml]
+----
+  <plugin>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-maven-plugin</artifactId>
+      <configuration>
+          <skip>true</skip>
+      </configuration>
+  </plugin>
+----
+
+==== Cache server
+
+The cache server is defined as follows
+
+.src/test/resources/test-cacheserver.yml
+----
+apiVersion: v1
+kind: List
+items:
+  - apiVersion: extensions/v1beta1
+    kind: Deployment
+    metadata:
+      name: cache-server
+      labels:
+        app: cache-server
+    spec:
+      replicas: 1
+      strategy:
+        type: Recreate
+      template:
+        metadata:
+          labels:
+            app: cache-server
+        spec:
+          containers:
+            - env:
+              - name: DEFAULT_CACHE_EVICTION_MAX_ENTRIES
+                value: "10"
+              - name: DEFAULT_CACHE_EXPIRATION_LIFESPAN
+                value: "10000"
+              - name: INFINISPAN_CONNECTORS
+                value: hotrod
+              image: registry.access.redhat.com/jboss-datagrid-7/datagrid71-openshift:1.1
+              name: cache-server
+              ports:
+              - containerPort: 8778
+                protocol: TCP
+              - containerPort: 11211
+                protocol: TCP
+              - containerPort: 11222
+                protocol: TCP
+              - containerPort: 11333
+                protocol: TCP
+              - containerPort: 8080
+                protocol: TCP
+              - containerPort: 8443
+                protocol: TCP
+  - apiVersion: v1
+    kind: Service
+    metadata:
+      labels:
+        app: cache-server
+      name: cache-server
+    spec:
+      selector:
+        app: cache-server
+      ports:
+      - name: 8080-tcp
+        port: 8080
+        protocol: TCP
+        targetPort: 8080
+      - name: 8443-tcp
+        port: 8443
+        protocol: TCP
+        targetPort: 8443
+      - name: 8778-tcp
+        port: 8778
+        protocol: TCP
+        targetPort: 8778
+      - name: 11211-tcp
+        port: 11211
+        protocol: TCP
+        targetPort: 11211
+      - name: 11222-tcp
+        port: 11222
+        protocol: TCP
+        targetPort: 11222
+      - name: 11333-tcp
+        port: 11333
+        protocol: TCP
+        targetPort: 11333
+      type: ClusterIP
+----
+
+=== Arquillian configuration
+
+With that cache server Openshift resource file in place, the only thing we need to do to make Arquillian use it as part of the environment setup is to add:
+
+`<property name="env.dependencies">file://${basedir}/target/test-classes/test-cacheserver.yml</property>`
```

Comment:
Same a previously: explain how to get the file instead of showing it and explain why we use the version in `target`.